DependencyInjection/HttplugExtension.php

```diff
@@ -207,4 +216,25 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Verify that Puli is installed
+     *
+     * @param ContainerBuilder $container
+     * @throws \Exception
+     */
+    private function verifyDiscoveryInstalled(ContainerBuilder $container)
+    {
+        $enabledBundles = $container->getParameter('kernel.bundles');
```

Comment:
I don't think we need to check against the bundle, we can check against the service as well. I think that is easier to mock.